packages/cerebral-demo/public/css/font-awesome.min.css

```diff
@@ -0,0 +1,4 @@
+/*!
```

Comment:
try to add `<script src="https://use.fontawesome.com/6103601291.js"></script>` into hml head to avoid commiting binaries.